test/spec/Feature/Query/PlanSpec.hs

```diff
@@ -192,7 +192,7 @@ spec actualPgVersion = do
       liftIO $ do
         resHeaders `shouldSatisfy` elem ("Content-Type", "application/vnd.pgrst.plan+json; for=\"application/json\"; charset=utf-8")
         resStatus `shouldBe` Status { statusCode = 200, statusMessage="OK" }
-        totalCost `shouldBe` 1.29
+        totalCost `shouldBe` 3.55
```

Comment:
I was wondering why it only changed for single upserts and not multiple ones. The `resolution=merge-duplicates` was not being applied in these tests, I commented in that PR (may need to be fixed in a new one): https://github.com/PostgREST/postgrest/pull/3005/files#r1366092283

Good news is the costs are still constant.